app/models/concerns/environmental.rb

```diff
@@ -61,15 +61,18 @@ module Environmental
     #
     # Use :default to automatically configure a default environment for the model.
     # Accepts a proc that resolves into an Environment or environment ID.
-    def has_environment(default: nil)
-      belongs_to :environment,
-        optional: true
+    def has_environment(default: nil, **kwargs)
+      belongs_to :environment, optional: true, inverse_of: nil, **kwargs
```

Comment:
I thought setting `inverse_of: nil` would resolve our issue but doesn't look like it did. We should rip this out until the possible bug[^1] in Rails is resolved.

[^1]: https://github.com/keygen-sh/keygen-api/pull/843/files#r1605536192